src/components/TabQueryEditor.vue

```diff
@@ -357,15 +369,28 @@
           const query = this.deparameterizedQuery
           this.$modal.hide('parameters-modal')
 
-          const runningQuery = this.connection.query(query)
+          this.runningQuery = this.connection.query(query)
           const queryStartTime = +new Date()
-          const results = await runningQuery.execute()
+          const results = await this.runningQuery.execute()
           const queryEndTime = +new Date()
           this.executeTime = queryEndTime - queryStartTime
           let totalRows = 0
           results.forEach(result => {
             result.rowCount = result.rowCount || 0
 
+            let rows = []
+            result.rows.forEach(row => {
+              const tmpRow = {}
+              for (let [key, value] of Object.entries(row)) {
+                if(value instanceof Buffer) {
```

Comment:
See data_mutators.js and add an entry there!

Reply:
If I'm not using that in query results yet, I should be

Reply:
I will try it again, im sure i tried it and the tabulator casted the value to a string.
A buffer casting to a string results to a json (see the screenshot in the issue).

i will report if i can do that with a mutator.

Reply:
Works and is committed.